circle.yml

```diff
@@ -1,38 +1,151 @@
-machine:
-  services:
-    - docker
-    - redis
-  node:
-    version:
-      6.9.1
-dependencies:
-  override:
-    - pip install --upgrade setuptools
-    - pip install -r requirements_dev.txt
-    - pip install -r requirements.txt
-    - npm install
-    - npm run build
-  cache_directories:
-    - node_modules/
-test:
-  override:
-    - pytest --junitxml=$CIRCLE_TEST_REPORTS/junit.xml tests/
-deployment:
-  tarball:
-    branch: [master, /release.*/]
-    commands:
-      - bin/pack
-  docker:
-    tag: /v[0-9]+(\.[0-9\-a-z]+)*/
-    commands:
-      - bin/pack
-      - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
-      - docker build -t redash/redash:$(./manage.py version | sed -e "s/\+/./") .
-      - docker push redash/redash:$(./manage.py version | sed -e "s/\+/./")
+version: 2.0
 notify:
   webhooks:
     - url: https://webhooks.gitter.im/e/895d09c3165a0913ac2f
-general:
-  branches:
-    ignore:
-      - gh-pages
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6
```

Comment:
We were using Node v6.9 because that's the latest that was available in the old platform. Now I assume we can use something newer?

Reply:
I didn't specify an exact version in error, but It defaults to 6.14 if I remember correctly. You can view some builds/logs here: https://circleci.com/gh/mozilla/redash/tree/update-circleci

Reply:
@arikfr There are a lot more modern npm versions on https://hub.docker.com/r/circleci/node/tags/ but I would recommend not doing an update at the same time as this pull request to remove the chance for conflicting issues.

Reply:
@jezdez I don't think 6.9 is available as a docker image though. Unless you are talking about doing a major update to the code-base, outside of this PR.

Reply:
Let's update to `circleci/node:8`, which is the latest LTS and the version I've been working with for along time now, so doubt there will be any issues.